velox/docs/functions/presto/conversion.rst

```diff
@@ -430,6 +448,23 @@ Invalid examples
 
   SELECT cast(214748364890 decimal(12, 2) as integer); -- Out of range
 
+From interval year month
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Casting from interval year month to integer returns the total number of months in the interval.
+The conversion is straightforward where 1 year = 12 months.
+
+Valid examples:
+
+::
+
+  SELECT cast(INTERVAL '1' YEAR as integer); -- 1
+  SELECT cast(INTERVAL '2' YEAR as integer); -- 2
+  SELECT cast(INTERVAL '1' MONTH as integer); -- 1
+  SELECT cast(INTERVAL '1-2' YEAR TO MONTH as integer); -- 14
+  SELECT cast(INTERVAL '-1' YEAR as integer); -- -1
+  SELECT cast(INTERVAL '-1-2' YEAR TO MONTH as integer); -- -14
+
```

Comment:
Since this cast is not supported in Presto, we may not update the doc for Presto conversion.

> presto> SELECT cast(INTERVAL '1' YEAR as integer);
Query 20250509_081817_00002_h6ney failed: line 1:8: Cannot cast interval year to month to integer